### ISSUE TITLE
Why require a zipped file?

### ISSUE THREAD

**Issue by cbou** [DivanteLtd/anonymizer#38]
It seems that a zipped file is required in order to anonymize the data.

Is correct?

Would it be possible to also accept pure sql files?

**Comment by mkoszut** [comment]
Yes, it is correct. It is historical reason, when I started write this tool I had database dumps in gzip format.

It is very easy to create gzipped file. On unix system you can use command `gzip name.sql`

**Comment by fbuchmeier** [comment]
I second the option to input plain text files. I wan to anonymise SQL dumps with 10GBs and more and un/zipping them takes quite some resources.
